node-express/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @navikt/team-soknad
```

Comment:
Put `/node-express/ @navikt/team-soknad` in the root `CODEOWNERS` instead

Reply:
men, får vi spesifisert at team søknad har ansvar for denne da?

Reply:
Yes, the path `/node-express/` instead of asterisk addresses this issue.